### ISSUE TITLE
Chapter 4, Table 4.2

### ISSUE THREAD

**Issue by jmg999** [awsccp/awsccp.github.io#12]
In Chapter 4, Understanding the AWS Environment, page 56, table 4.2 is incorrect.  The preceding sentence is correct, however.  It states, "Private networks--including AWS subnets--using the IPv4 protocol are allowed to use all the addresses w/in the three address ranges shown in Table 4.2.  Table 4.2, however, fails to list all three address ranges, instead leaving out 10.0.0.0-10.255.255.255 and its 16,777,216 available, private IP addresses.  I've attached an image of the table to this errata notification.

![Table 4 2](https://user-images.githubusercontent.com/33406288/79689460-ce8ec100-8209-11ea-8fe8-63633cabe530.png)

**Comment by dbclinton** [comment]
You're absolutely right on this. I'm not sure how the 10.0.0.0 range was dropped off (it's there in my earlier manuscript versions), but it's definitely not in the book. I'll make sure this is corrected.
Thanks!
David
